stl/inc/format

```diff
@@ -137,15 +137,15 @@ public:
     // _Next_arg_id > 0 means automatic
     // _Next_arg_id == 0 means unknown
     // _Next_arg_id < 0 means manual
-    constexpr size_t next_arg_id() {
+    _NODISCARD constexpr size_t next_arg_id() {
         if (_Next_arg_id < 0) {
             throw format_error("Can not switch from manual to automatic indexing");
         }
 
         return static_cast<size_t>(_Next_arg_id++);
     }
 
-    constexpr void check_arg_id(const size_t _Id) {
+    _NODISCARD constexpr void check_arg_id(const size_t _Id) {

```

Comment:
We conventionally a _void_ `_NODISCARD` here

Reply:
Whoops, I didn't mean to add `_NODISCARD` to this one. But I am not really sure what you mean.

Reply:
@CaseyCarter will know

Reply:
He means that `[[nodiscard]]` doesn't make sense on `void`-returning functions, and he's saying so via clever wordplay (`void` / avoid).

Reply:
Oh haha, I see it now. My brain totally missed the 'a'.